src/erlcloud_cloudtrail.erl

```diff
@@ -192,8 +192,3 @@ port_spec(#aws_config{cloudtrail_port=80}) ->
 port_spec(#aws_config{cloudtrail_port=Port}) ->
     [":", erlang:integer_to_list(Port)].
 
-
-timeout(#aws_config{timeout = undefined}) ->
-    1000;
```

Comment:
good spotting. addressed. 
in future I would really like to understand why DDB and CT are that different and the actual reason for that.